public/docs/ts/latest/guide/dependency-injection.jade

```diff
@@ -890,7 +890,7 @@ block dart-map-alternative
   An `Injector` is itself an injectable service.
 
   In this example, Angular injects the component's own `Injector` into the component's constructor.
-  The component then asks the injected injector for the services it wants.
+  The component then asks the injected injector for the services it wants on `ngOnInit`.
```

Comment:
OK but better would be "in `ngOnInit()`. Use  "in" instead of "on"

Kapunahele has started highlighting method names by putting parens after them. I think it's a good idea. We'll have a lot of prose to fix if we follow her lead. We can start any time :)